pkg/monitor/monitor.go

```diff
@@ -280,32 +298,76 @@ func (m *Monitor) cleanupDriverNodePods(node *volume.NodeInfo) {
 		log.Errorf("Error cleaning up volume attachments: %v", err)
 	}
```

Comment:
Some optimisations can be done inside cleanupVolumeAttachmentsByNode().
vaList, err := storage.Instance().ListVolumeAttachments() is being called every time. There is scope of calling this once and use it for the current iteration of the driver monitor.

Reply:
Also isSameNode() is being called inside the same function which calls core.Instance().GetNodeByName() which is an unnecessary API call. We have a K8s node name list in the map already. We could either create a map of name to K8s node object and use it. Or just iterate over it every time. This would avoid making an extra api call. Let me know what you think.

Reply:
As discussed not addressing the first comment of doing `ListVolumeAttachments` once as we are going to limit the number of goroutines.

Number of GetNodeByName is replaced with doing one GetNode call and create the map and use it for one cycle of handling offline nodes in drivermonitor.